src/object_based/extent.cpp

```diff
@@ -177,17 +212,31 @@ namespace ear {
     return positions;
   }
 
-  Eigen::MatrixXd SpreadingPanner::_generatePanningPositionsResults() {
-    Eigen::MatrixXd results(_panningPositions.rows(),
-                            _psp->numberOfOutputChannels());
-    for (int i = 0; i < _panningPositions.rows(); ++i) {
-      results.row(i) = _psp->handle(_panningPositions.row(i)).get();
+  Eigen::MatrixXd SpreadingPannerBase::_generatePanningPositionsResults(
```

Comment:
Does this get called much? Copying std::shared_ptr is relatively expensive and it looks like it's not stored in this function so maybe pass the panner by ref/const ref instead

Reply:
Oh just realised it's only called from the panner constructor so probably not an issue

Reply:
Yeah, the performance isn't an issue, but `const&` is simpler and shorter, so I might as well change it.